app/frontend/wallet/shelley/helpers/addresses.ts

```diff
@@ -27,6 +27,10 @@ export const accountAddressFromXpub = (stakeXpub: Xpub, networkId) => {
   return bech32.encode({prefix: 'addr', data: addrBuffer})
 }
 
+export const accountHexAddressFromXpub = (stakeXpub: Xpub, networkId) => {
```

Comment:
I would annotate the return type as it is not clear at first glance - is it really a (hex)string or a buffer? I would do the same for the surrounding functions